web_graph_radar/README.md

```diff
@@ -0,0 +1,4 @@
+Graph Radar Chart.
+=================
+
+Add new graph view mode: Radar
```

Comment:
Then add there to Contributors yourself and 

     Florent THOMAS (financial support) <https://github.com/MindAndGo>